src/representation_theory/maschke.lean

```diff
@@ -136,22 +136,32 @@ end
 end linear_map
 end
 
+namespace char_zero
+
+variables {k : Type u} [field k] {G : Type u} [fintype G] [group G] [char_zero k]
+
+instance : fact ¬(ring_char k ∣ fintype.card G) :=
+⟨by simp [fintype.card_eq_zero_iff]⟩
```

Comment:
No global `fact`s are allowed. You can make this a def, and then use `localized` to enable it in some locale.

Reply:
Alternate plan: how do you feel about having everything depend instead on `invertible (fintype.card G : k)`, which can just be a global instance?

Reply:
Sure!